pkg/cmd/codespace/create.go

```diff
@@ -38,6 +39,7 @@ func newCreateCmd(app *App) *cobra.Command {
 
 	createCmd.Flags().StringVarP(&opts.repo, "repo", "r", "", "repository name with owner: user/repo")
 	createCmd.Flags().StringVarP(&opts.branch, "branch", "b", "", "repository branch")
+	createCmd.Flags().StringVarP(&opts.location, "location", "l", "", "location (EastUs, SouthEastAsia, WestEurope, or WestUs2). Assigned by IP or user settings if not provided")
```

Comment:
We could use the syntax CLI uses for mutually exclusive values:
```suggestion
	createCmd.Flags().StringVarP(&opts.location, "location", "l", "", "location: {EastUs|SouthEastAsia|WestEurope|WestUs2} (determined automatically if not provided)")
```